MAUI/ListView/emptyview.md

```diff
@@ -0,0 +1,320 @@
+The `SfListView.EmptyView` can be set to custom type which is used to display when the `ItemsSource` is null, or when the collection specified by the `ItemsSource` property is null or empty. The appearance of the `EmptyView` can be customized by using `SfListView.EmptyViewTemplate`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView"
+             xmlns:local="clr-namespace:EmptyViewTemplate">
+  <Grid>
+    <Grid.RowDefinitions>
+      <RowDefinition Height="30"/>
+      <RowDefinition Height="*" />
+    </Grid.RowDefinitions>
+    <SearchBar x:Name="filterText" 
+               FontSize="16"                              
+               Placeholder="Filter Inventory" TextChanged="SearchBar_TextChanged"/>                                                   
+    <syncfusion:SfListView Grid.Row="1" x:Name="listView"
+                           ItemsSource="{Binding Items}"
+                           ItemSize="56">
+      <syncfusion:SfListView.EmptyView>
+        <local:FilterItem Filter="{Binding Source={x:Reference filterText},Path=Text}"/>
+      </syncfusion:SfListView.EmptyView>
+      <syncfusion:SfListView.EmptyViewTemplate>
+        <DataTemplate>
+          <Label Text="{Binding Filter,StringFormat='{0} is not found'}" HorizontalTextAlignment="Center"     
+                 VerticalOptions="CenterAndExpand"
+                 FontSize="18" FontFamily="Roboto-Regular"/>
+        </DataTemplate>
+      </syncfusion:SfListView.EmptyViewTemplate>                         
+    </syncfusion:SfListView>
+  </Grid>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.EmptyView = new FilterItem() { Filter = filterText.Text};
+
+listView.EmptyViewTemplate = new DataTemplate(() =>
+{
+  Label label = new Label()
+  {					
+    FontSize = 18,
+    FontFamily = "Roboto-Regular",
+    VerticalOptions = LayoutOptions.CenterAndExpand,
+    HorizontalTextAlignment = TextAlignment.Center
+  };
+  label.SetBinding(Label.TextProperty, new Binding("Filter", stringFormat : "{0} is not found"));
+  return label;
+});
+{% endhighlight %}
+{% endtabs %}
+
+The `FilterItem` type defines a `Filter` property.
+
+{% tabs %}
+{% highlight c# tabtitle="FilterItem.cs" %}
+public class FilterItem : BindableObject
+{
+  public static readonly BindableProperty FilterProperty = BindableProperty.Create(nameof(Filter), typeof(string), typeof(FilterItem), null);
+
+  public string Filter
+  {
+      get { return (string)GetValue(FilterProperty); }
+      set { SetValue(FilterProperty, value); }
+  }
+}
+{% endhighlight %}
+{% endtabs %}
+
+FilterItem object is set to `SfListView.EmptyView` property and `Filter` property is binds to `SearchBar.Text` property. When SearchBar.TextChanged event is raised, value of `SearchBar.Text` property is stored in `Filter` property.   

```

Comment:
FilterItem object is set to the `SfListView.EmptyView` property, and the `Filter` property is bound to the `SearchBar.Text` property. When SearchBar.TextChanged event is raised, the value of the `SearchBar.Text` property is stored in the `Filter` property.